package.json

```diff
@@ -12,6 +12,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "^2.1.3",
+    "styled-components": "^5.3.5",
```

Comment:
???

Reply:
It need for change theme from dark to light and vice versa